modules/customizing-the-appearance/proc-customize-rhdh-sidebar-menuitems.adoc

```diff
@@ -1,129 +1,60 @@
 [id='proc-customize-rhdh-sidebar-menuitems_{context}']
 = Customizing the sidebar menu items for your {product-short} instance
 
-The sidebar menu in {product} consists of two main parts:
+The sidebar menu in {product} consists of two main parts that you can configure:
 
-* *Main menu items*: These items are the static menu items that form the core navigation structure of sidebar. These menu items remain consistent and are predefined.
-
-* *Dynamic plugin menu items*: These items are displayed beneath the main menu and can be customized based on the plugins installed. The main menu items section is dynamic and can change based on your preferences and installed plugins.
+* *Dynamic plugin menu items*: Your preferences and your active plugins define dynamically one part of the sidebar menu.
+* *Main menu items*: The core navigation structure of sidebar is static.
```

Comment:
Consider using description lists, which would be semantically correct in the context. See https://docs.asciidoctor.org/asciidoc/latest/lists/description/

```suggestion
Dynamic plugin menu items:: Your preferences and your active plugins define dynamically one part of the sidebar menu.
Main menu items:: The core navigation structure of sidebar is static.
```